vars/setupCommonPipelineEnvironment.groovy

```diff
@@ -97,6 +101,7 @@ void call(Map parameters = [:]) {
         Map config = ConfigurationHelper.newInstance(this)
             .loadStepDefaults()
             .mixinGeneralConfig(script.commonPipelineEnvironment, GENERAL_CONFIG_KEYS)
+            .mixin(parameters, PARAMETER_KEYS)
```

Comment:
Why was this introduced?

Reply:
Since it was missing. Without this line the parameters handed over via signature (`parameters`) would not be contained in the configuration. In that case we could access these parameters below via `parmeters.gitUrl` but not like `config.gitUrl` which is the more common way in all the other steps.

Reply:
got it